### PR TITLE
UXG-1270 add listing guided setup page

### PR DESCRIPTION
**Pull request by nicmarson**
http://jira.hq.idx.org/browse/UXG-1270

### CODE REVIEW

File: src/vue/backend/src/views/guided-setup/Listings.vue [hunk 1]
```diff
@@ -0,0 +1,100 @@
+<template>
+    <GuidedSetupContentCard
+        :cardTitle="cardTitle"
+        :steps="guidedSetupSteps"
```
Comment (sgreenl5) [inline src/vue/backend/src/views/guided-setup/Listings.vue:4]: The steps are not displaying their icons

File: src/vue/backend/src/views/guided-setup/Listings.vue [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        </template>
+        <template v-slot:controls>
+            <p>Activating IMPress Listings will:</p>
+            <idx-list customClass="list-featured">
+                <idx-list-item>Feature 1</idx-list-item>
+                <idx-list-item>Feature 2</idx-list-item>
+                <idx-list-item>Feature 3</idx-list-item>
+                <idx-list-item>Feature 4</idx-list-item>
+            </idx-list>
+            <idx-block className="form-content">
```
Comment (sgreenl5) [inline src/vue/backend/src/views/guided-setup/Listings.vue:20]: The borders are too dark, using #E4E4E4 but the design is #EFEFEF

File: src/vue/backend/src/views/guided-setup/Listings.vue [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+import { mapState, mapActions } from 'vuex'
+import GuidedSetupContentCard from '@/templates/GuidedSetupContentCard.vue'
+export default {
+    components: {
+        GuidedSetupContentCard
+    },
+    data () {
+        return {
+            cardTitle: 'Activate IMPress Listings',
+            activateLabel: 'Activate',
```
Comment (sgreenl5) [inline src/vue/backend/src/views/guided-setup/Listings.vue:48]: @ralls Are we wanting to update the verbiage to "Enable" Across the board?

File: src/vue/backend/src/views/guided-setup/Listings.vue (hunk 1 above)
Comment (nicmarson) [reply src/vue/backend/src/views/guided-setup/Listings.vue:4]: yeah, looks like the sprite got removed from `page` so I'll add it back in.

File: src/vue/backend/src/views/guided-setup/Listings.vue (hunk 3 above)
Comment (ralls) [reply src/vue/backend/src/views/guided-setup/Listings.vue:48]: Yes but we can either do that now or when we decouple state from form data

**Review by sgreenl5 — APPROVED** [review]

**Review by ralls — APPROVED** [review]
👍

**Pull request merged**